packages/adapter-knex/lib/adapter-knex.js

```diff
@@ -203,10 +224,20 @@ class KnexAdapter extends BaseKeystoneAdapter {
 
   // This will completely drop the backing database. Use wisely.
   dropDatabase() {
-    const tables = Object.values(this.listAdapters)
-      .map(listAdapter => `"${this.schemaName}"."${listAdapter.tableName}"`)
-      .join(',');
-    return this.knex.raw(`DROP TABLE IF EXISTS ${tables} CASCADE`);
+    if (this.isClientPostgres) {
+      const tables = Object.values(this.listAdapters)
+        .map(listAdapter => `"${this.schemaName}"."${listAdapter.tableName}"`)
+        .join(',');
+      return this.knex.raw(`DROP TABLE IF EXISTS ${tables} CASCADE`);
+    } else if (this.isClientMySQL) {
+      const tableNames = Object.values(this.listAdapters).map((listAdapter) => listAdapter.tableName);
+      return this.knex.transaction(async (transaction) => {
+          await transaction.raw('SET FOREIGN_KEY_CHECKS = 0');
+          await transaction.raw(`DROP TABLE IF EXISTS ${tableNames.join(',')}`);
+          return transaction.raw('SET FOREIGN_KEY_CHECKS = 1');
+      });
+    }
+    return Promise.reject(new Error(`KnexAdapter.dropDatabase does not support the '${this.client}' client.`));
```

Comment:
This should be a `throw`.

```suggestion
    throw new Error(`KnexAdapter.dropDatabase does not support the '${this.client}' client.`);
```